### PR TITLE
fix(api): update count if dont convert

### PR DESCRIPTION
**Pull request by Orta21**
Ref. metriport/metriport-internal#799

Ref: #799

### Description

Issue in sandbox when querying docs that the convert stalls.

For reference https://metriport.slack.com/archives/C0627AB65AB/p1698842626026559

### Release Plan

asap

- [ ] merge once approved

### CODE REVIEW

**Review by RamilGaripov — APPROVED** [review]
Approving, assuming this works. So the logic here is that if one of the documents isn't converted, then we set the convertProgress to undefined. How does this help trigger a conversion for that document?

**Comment by Orta21** [comment]
> Approving, assuming this works. So the logic here is that if one of the documents isn't converted, then we set the convertProgress to undefined. How does this help trigger a conversion for that document?

Actually ill need to update this but the idea is if we dont convert what we were suppose to then we need to set it to complete

File: packages/api/src/external/commonwell/document/document-query-sandbox.ts [hunk 1]
```diff
@@ -142,7 +147,10 @@ export async function sandboxGetDocRefsAndUpsert({
       status: "completed",
       successful: entries.length,
     },
-    convertProgress: undefined,
+    convertProgress: docsToConvert === 0 ? {
```
Comment (leite08) [inline packages/api/src/external/commonwell/document/document-query-sandbox.ts:150]: Safe programming: `docsToConvert <= 0 ?...`

**Review by leite08 — COMMENTED** [review]
leaving a comment for now, waiting for the final update

**Review by leite08 — APPROVED** [review]

**Pull request merged**